resources/user-agents/browsers/puffin/puffin-5-0-on.json

```diff
@@ -101,24 +101,28 @@
           ]
         },
         {
-          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/*Safari/* Puffin/#MAJORVER#.#MINORVER#.*IP*",
+          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/*Safari/* Puffin/#MAJORVER#.#MINORVER#.*I?*",
           "devices": {
             "iPhone": "iPhone",
             "iPad": "iPad"
           },
           "engine": "WebKit",
           "platforms": [
-            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2",
-            "iOS_A"
+            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2", "iOS_A_10_3",
+            "iOS_A",
+            "iOS_C_10_0", "iOS_C_10_1", "iOS_C_10_2", "iOS_C_10_3",
+            "iOS_C"
           ]
         },
         {
           "match": "Mozilla/5.0 (#PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/*Safari/* Puffin/#MAJORVER#.#MINORVER#.*I?*",
           "device": "general Mobile Device (Apple)",
           "engine": "WebKit",
           "platforms": [
-            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2",
-            "iOS_A"
+            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2", "iOS_A_10_3",
+            "iOS_A",
+            "iOS_C_10_0", "iOS_C_10_1", "iOS_C_10_2", "iOS_C_10_3",
```

Comment:
This should have no effect for browscap-php, but maybe for other clients, but normally we dont care about them.

Some time ago (maybe before browscap-php v3) I wanted to change the sorting in the ini file, that the client does not need to resort anything. But this did not work.